package.json

```diff
@@ -31,7 +31,7 @@
     "ember-cli-babel": "^6.0.0",
     "ember-cli-is-package-missing": "^1.0.0",
     "ember-cookies": "^0.0.13",
-    "ember-fetch": "^2.1.0",
+    "ember-fetch": "2.1.0 - 3",
```

Comment:
I'd prefer `^2.1.0 || ^3.0.0` but a range constraint would probably work too. The only bad thing about the range constraint is that it looks as if `3.0.1` wasn't covered by it, even though it actually is.